README.md

```diff
@@ -25,18 +25,87 @@ Designed to be container for repeatedly used set of constants.
 
 **Example**:
 ```php
-class Gender extends Saritasa\Enum
+class LogicOperations extends Saritasa\Enum
 {
-    const MALE = 'Male';
-    const FEMALE = 'Female';
+    protected const AND = null;
+    protected const OR = null;
+    protected const XOR = null;
 }
 ```
-then somewere in code:
+then somewhere in code:
 ```php
-$allGenders = Gender::getConstants();
-$gender = new Gender($stringValue); // Will throw UnexpectedValueException on unknown value;
-function getGenderDependentValue(Gender $gender) { ... }
+$operations = LogicOperations::getConstantNames(); // returns ['AND', 'OR', 'XOR']
+...
+function getLogicOperationDependentValue(LogicOperations $op) 
+{
+    if ($op == LogicOperations::OR()) { ... }
+    ...
+    switch ($op) {
+        case LogicOperations::AND():
+            ....
+            break;
+        case LogicOperations::OR():
+            ...
+            break;
+        case LogicOperations::XOR():
+            ...
+            break;
+        default:
+            ...
+    }
+}
+...
+$xor = LogicOperations::XOR(); // will throw InvalidEnumValueException on unknown value
```

Comment:
Why this throws an exception if XOR constant defined in LogicOperations enum?

Reply:
This is old comment. I'll add new code line with non-existing constant.